content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -174,7 +174,7 @@ different Kubernetes components.
 | `TokenRequestProjection` | `false` | Alpha | 1.11 | 1.11 |
 | `TokenRequestProjection` | `true` | Beta | 1.12 | |
 | `TTLAfterFinished` | `false` | Alpha | 1.12 | |
-| `VolumePVCDataSource` | `false` | Alpha | 1.15 | |
+| `VolumePVCDataSource` | `true` | Beta | 1.16 | |
```

Comment:
I think we add a new line for beta and keep the alpha entry

Reply:
Oh shoot, you're right fixed.